README.rst

```diff
@@ -27,8 +27,22 @@ BigchainDB Python Driver
 Features
 --------
 
-* Minimal support for `CREATE`, `TRANSFER`, and `GET` operations on the
-  `/transactions` endpoint.
+* Support for preparing, fulfilling, and sending transactions to a BigchainDB
+  node.
+* Retrieval of transactions by id.
+* Getting status of a transaction by id.
+
+
+Compatibility Matrix
+--------------------
+
++-----------------------+-----------------------+
+| **BigchainDB Server** | **BigchainDB Driver** |
++=======================+=======================+
+| ``0.7.x``             | ``0.0.3``             |
++-----------------------+-----------------------+
+| ``master``            | ``master``            |
```

Comment:
@sbellem I realize you already merged this pull request, but it occurred to me that it might be overly optimistic to claim that the two `master` branches are always compatible with each other. Usually what happens is the server gets ahead of the Python driver, and then the Python driver catches up.